src/ipc/ipc4/helper.c

```diff
@@ -147,24 +147,23 @@ struct ipc_comp_dev *ipc_get_comp_by_ppl_id(struct ipc *ipc, uint16_t type, uint
 	return NULL;
 }
 
-static int ipc4_create_pipeline(uint32_t pipeline_id, uint32_t priority,
-				uint32_t memory_size, uint32_t core_id)
+static int ipc4_create_pipeline(struct ipc4_pipeline_create *pipe_desc)
```

Comment:
looks like it can be `const`?

Reply:
good catch - I will change it to const in next patch. Thanks